mcmc/MCMCProcessor.h

```diff
@@ -52,7 +52,7 @@
 //KS: Enum for different covariance classes
 enum ParameterEnum {
   kXSecPar  = 0, //KS: This hold both xsec and flux
-  kND280Par = 1,
+  kNDPar = 1,
```

Comment:
Be aware that T2K code used kND280Par temporarily
for example here
https://github.com/t2k-software/MaCh3/blob/e287d9eb4de2573455fb80b1dac68d7c122b3916/src/ND280_MCMC.cpp#L67

so be prepared for compilation issues